parsing/schools/neu/courses.py

```diff
@@ -0,0 +1,64 @@
+# Copyright (C) 2017 Semester.ly Technologies, LLC
+#
+# Semester.ly is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Semester.ly is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+
+
+from __future__ import absolute_import, division, print_function
+
+import re
+import sys
+import os
+import subprocess
+
+from bs4 import NavigableString, Tag
+
+from parsing.library.base_parser import BaseParser
+from parsing.library.exceptions import ParseError
+from parsing.library.utils import safe_cast, dict_filter_by_dict
+from semesterly.settings import get_secret
+
+
+class Parser(BaseParser):
+   
+    def __new__(cls, *args, **kwargs):
```

Comment:
i know this is in the template but if you dont have any secrets to load, no need to have a `__new__` method

Reply:
Removed